std/range/package.d

```diff
@@ -2887,13 +2905,37 @@ private:
     UT _value;
 
 public:
-    @property inout(T) front() inout { return _value; }
-    @property inout(T) back() inout { return _value; }
+    ///
+    @property inout(T) front() inout
+    {
+        return _value;
+    }
+
+    ///
+    @property inout(T) back() inout
+    {
+        return _value;
+    }
+
+    ///
     enum bool empty = false;
+
+    ///
     void popFront() {}
+
+    ///
     void popBack() {}
+
+    ///
     @property auto save() inout { return this; }
```

Comment:
Why did you expand all the other primitives, but not this one? And why expand them at all? Is there a rule on this in the style guidelines? I prefer the original way better.